### PR TITLE
kubevirtci: Add alpine provision presubmit job

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Enrique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml [hunk 1]
```diff
@@ -323,3 +323,44 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
```
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:326]: can we use `run_if_changed` to narrow down when there are changes on the image setup?

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml [hunk 2]
```diff
@@ -323,3 +323,44 @@ presubmits:
...
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - cd cluster-provision/images/vm-image-builder && ./create-containerdisk.sh alpine-cloud-init
+        image: quay.io/kubevirtci/golang:v20220211-d7d6c59
+        name: ""
+        resources:
+          requests:
+            memory: 8Gi
```
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:347]: Do we really need 8Gi?

**Review by dhiller — CHANGES_REQUESTED** [review]

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml (hunk 1 above)
Comment (qinqon) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:326]: Then rehearse will not test this, I think is not worth it.

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml (hunk 2 above)
Comment (qinqon) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:347]: Nope 1Gi or even 500Mb are enough

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml (hunk 1 above)
Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:326]: OK, as it uses less resources now, it should be fine.

**Review by dhiller — APPROVED** [review]
/approve

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml [hunk 3]
```diff
@@ -323,3 +323,44 @@ presubmits:
...
+  - always_run: true
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      timeout: 0h10m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+    max_concurrency: 1
+    name: check-provision-alpine-container-disk
```
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml:335]: Can you rename the job to make it more clear? There's already an alpine container disk in existence.
```suggestion
    name: check-provision-alpine-cloud-init-container-disk
```

**Review by dhiller — COMMENTED** [review]
I think we should change the job name to make it more clear.

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml (hunk 3 above)
Comment (qinqon) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml]: using "with-test-tooling" we have a similar one for fedora

**Review by dhiller — APPROVED** [review]
/unhold

**Pull request merged**